package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Front-End Capstone Project",
   "main": "server/index.js",
   "scripts": {
-    "build": "webpack",
-    "serve": "nodemon",
+    "build": "webpack --config webpack.config.js -w",
+    "serve": "nodemon server/index.js",
```

Comment:
This looks good, but it was working without the change for me. Matt may be able to weight in more on what this config is doing without the server/index.js added. I'm cool either way.